src/Countries.php

```diff
@@ -258,6 +258,7 @@ class Countries implements \Iterator, \ArrayAccess
         'VI' => 'Virgin Islands, U.S.',
         'WF' => 'Wallis And Futuna',
         'EH' => 'Western Sahara',
+        'XI' => 'Northern Ireland',
```

Comment:
Xi is not a valid IsoCode cf https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

It's only used for VAT but it's not an isocode.